.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Environment information**
+* Malwarecage version (from `/about`):
+* Installation method:
+  - [ ] Generic docker-compose
+  - [ ] Dev docker-compose
+  - [ ] Native (from source)
+  - [ ] Other (please explain)
+
+**Reproduction Steps**
+
+<!-- Please explain how can we reproduce your problem. -->
+
+
+**Expected behaviour**
+
+<!-- What did you expect to happen? -->
+
+
+**Actual behaviour the bug**
+
+<!-- What actually happened? -->
```

Comment:
I think that the logical steps here are reversed. When I want to open a bug report I want to -

First, describe what is the bug I experience (what happened)
Then, I want to describe what I think should've happened - what is my expected behavior
Last, I can share the steps I did so you can reproduce